dev/reference/tools/data-migration/manage-tasks.md

```diff
@@ -901,6 +903,50 @@ purge-relay -w "127.0.0.1:8262" --filename "mysql-bin.000003"
 >
 > 一般不需要使用此命令。仅当已确定 `task => DM-workers` 映射关系存在，但执行其它命令时仍提示必须刷新它时，你才需要使用此命令。
 
+## dmctl 基本用法--命令模式
+
+命令模式跟交互模式的区别是，执行时只需要在 dmctl 命令后紧跟任务操作执行，任务操作同交互模式参数一致。
+
+> **注意：**
+>
+> 1. 一次 dmctl 命令只能跟一个任务操作
+>
+> 2. 任务操作只能放在 dmctl 命令的最后
+
+{{< copyable "shell-regular" >}}
+
+```bash
+./dmctl -master-addr 172.16.30.14:8261 start-task task.yaml
+./dmctl -master-addr 172.16.30.14:8261 stop-task task
+./dmctl -master-addr 172.16.30.14:8261 query-status
+```
+
+```
+Available Commands:
```

Comment:
What about taking the `Available Commands` out of the code block?

Reply:
It's kind of output of dmctl, I think if fine put it here @csuzhangxc What do you think?

Reply:
Putting it in the code block is fine. (It's kind of output of dmctl)